BuildResidentialHPXML/measure.rb

```diff
@@ -3551,14 +3568,40 @@ def self.create_geometry_envelope(runner, model, args)
     return true
   end
 
-  def self.set_header(hpxml, args)
+  def self.unavailable_period_exists(hpxml, column_name, begin_month, begin_day, begin_hour, end_month, end_day, end_hour, natvent_availability = nil)
+    hpxml.header.unavailable_periods.each do |unavailable_period|
+      begin_hour = 0 if begin_hour.nil?
+      end_hour = 24 if end_hour.nil?
+
+      next unless (unavailable_period.column_name == column_name) &&
+                  (unavailable_period.begin_month == begin_month) &&
+                  (unavailable_period.begin_day == begin_day) &&
+                  (unavailable_period.begin_hour == begin_hour) &&
+                  (unavailable_period.end_month == end_month) &&
+                  (unavailable_period.end_day == end_day) &&
+                  (unavailable_period.end_hour == end_hour)
+      if unavailable_period.natvent_availability == natvent_availability
+        return true
+      end
```

Comment:
Why is natvent_availability handled differently? Isn't the below code the same?

```suggestion
      next unless (unavailable_period.column_name == column_name) &&
                  (unavailable_period.begin_month == begin_month) &&
                  (unavailable_period.begin_day == begin_day) &&
                  (unavailable_period.begin_hour == begin_hour) &&
                  (unavailable_period.end_month == end_month) &&
                  (unavailable_period.end_day == end_day) &&
                  (unavailable_period.end_hour == end_hour) &&
                  (unavailable_period.natvent_availability == natvent_availability)
        return true
      end
```